c++/src/capnp/compiler/capnp-test.sh

```diff
@@ -37,10 +39,10 @@ else
   CAPNP=${CAPNP:-capnp}
 fi
 
-SCHEMA=`dirname "$0"`/../test.capnp
-JSON_SCHEMA=`dirname "$0"`/../compat/json-test.capnp
-TESTDATA=`dirname "$0"`/../testdata
-SRCDIR=`dirname "$0"`/../..
+SCHEMA=$BASE/../test.capnp
```

Comment:
This part of the change looks unnecessary -- any particular reason for it? If not would you mind reverting it so that it's clear what's actually being fixed?